juce/loong_fix_relax.patch

```diff
@@ -0,0 +1,11 @@
+--- a/modules/juce_audio_devices/native/oboe/include/oboe/StabilizedCallback.h	2024-04-15 17:46:51.000000000 +0800
++++ b/modules/juce_audio_devices/native/oboe/include/oboe/StabilizedCallback.h	2024-11-27 19:40:34.884000000 +0800
+@@ -60,7 +60,7 @@
+ #if defined(__i386__) || defined(__x86_64__)
+ #define cpu_relax() asm volatile("rep; nop" ::: "memory");
+ 
+-#elif defined(__arm__) || defined(__mips__) || defined(__riscv)
++#elif defined(__arm__) || defined(__mips__) || defined(__riscv) || defined(__loongarch64_lp64)
```

Comment:
Other architectures here do not emphasize 64-bit, do we really need to emphasize it?

Reply:
Arch `aarch64` is also difined here below

`#elif defined(__aarch64__)`
`#define cpu_relax() asm volatile("yield" ::: "memory")`

Reply:
Why not `__loongarch__`

Reply:
It's OK, just I have no loongarch32 machine and I didn't know whether the *JUCE* should supports loong32

Reply:
> It's OK, just I have no loongarch32 machine and I didn't know whether the _JUCE_ should supports loong32

This is actually less important, it's just to fit the style

Reply:
adjusted